package.json

```diff
@@ -128,7 +128,7 @@
     },
     "dependencies": {
         "@0x/assert": "^3.0.4",
-        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-ae2a6fb68",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-766888ec4",
```

Comment:
⚠️ This updates to https://github.com/0xProject/0x-monorepo/pull/2646 ⚠️ 

Will need to modify this once 2646 is merged